embedded-hal-async/src/spi.rs

```diff
@@ -69,9 +173,13 @@ pub trait SpiDevice: ErrorType {
     /// On bus errors the implementation should try to deassert CS.
     /// If an error occurs while deasserting CS the bus error should take priority as the return value.
     ///
+    /// # Safety
```

Comment:
Mention that the `transaction!` safe wrapper macro exist and that using it if possible is recommended.